raiden/api/rest.py

```diff
@@ -84,8 +95,8 @@ def add_resource(self, resource_cls):
             resource_cls._route
```

Comment:
It took me some time to realize this is not part of the `flask`/`flask-restful` framework, do we really need to invent a new pattern here?